README.md

```diff
@@ -1,10 +1,10 @@
-# Docker Hub Image
+# Image Overview
 
-The files in this directory are used to build the `ibmcom/websphere-traditional` images on [Docker Hub](https://hub.docker.com/r/ibmcom/websphere-traditional/). These images contain the ILAN licensed IBM WebSphere Application Server traditional. For production use or if you wish to build these yourself just follow [these instructions](docker-build/9.0.5.x/README.md), otherwise please see below on how to extend our pre-built image with your application and configuration!  Once you're ready to deploy this into Kubernetes, please see our [helm chart](https://github.com/IBM/charts/tree/master/stable/ibm-websphere-traditional).
+The files in this directory are used to build the `websphere-traditional` images. You can see a list of images provided [here](/docs/images.md). These images contain the ILAN licensed IBM WebSphere Application Server traditional. For production use or if you wish to build these yourself just follow [these instructions](docker-build/9.0.5.x/README.md), otherwise please see below on how to extend our pre-built image with your application and configuration!
 
-**Try IBM Container Registry:**   Our traditional WebSphere Application Server [images](https://www.ibm.com/docs/en/was/9.0.5?topic=container-websphere-application-server-images) are also available from IBM Container Registry (ICR) at `icr.io/appcafe/websphere-traditional`.  Unlike with Docker Hub, images can be pulled from ICR without rate limits and without authentication. Only images with Universal Base Image (UBI) as the Operating System are available in ICR at the moment.
+**Registry Options**
 
-**Note:** The 9.0.5.3 release images were the last version where `latest` and `9.0.5.#` tagged images were based upon Ubuntu v16.04 as the base operating system.  Starting with the 9.0.5.4 release, the images are now based upon Red Hat Universal Base Image (UBI) 8 OS images.  As always, you are welcome to build your own images using any base OS image that supports IBM WebSphere Application traditional.
+You can pull the images built using this directory from both IBM Container Regsitry (icr.io) and [Docker Hub](https://hub.docker.com/r/ibmcom/websphere-traditional/). Unlike with Docker Hub, images can be pulled from IBM Container Regsitry without rate limits and without authentication.
```

Comment:
Remove these 2 references to Docker Hub:
1. https://github.com/WASdev/ci.docker.websphere-traditional/blob/604baa2404f6d0ed6849ae332aa6bdf13ab20ba4/README.md?plain=1#L7
You can try it today for free from [Docker Hub](https://hub.docker.com/_/websphere-liberty/) 
-->
You can try it today for free from icr.io. See [Liberty container images](https://www.ibm.com/docs/en/was-liberty/base?topic=images-liberty-container) for more information.

2. https://github.com/WASdev/ci.docker.websphere-traditional/blob/604baa2404f6d0ed6849ae332aa6bdf13ab20ba4/docs/images.md?plain=1#L3
Our recommendation is to use ICR instead of Docker Hub since ICR doesn't impose rate limits on image pulls --> Our recommendation is to use ICR as it doesn't impose rate limits on image pulls